### PR TITLE
fix(Settings): Fix hover animation of night mode modal button

### PR DESCRIPTION
**Pull request by vkWeb**
### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of freeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.org/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

### Checklist:

- [x] Tested changes locally.
- [ ] Addressed currently open issue (replace XXXXX with an issue no in next line)

### Description
My recent PR #16501 broke the hover animation of danger-button in night mode modals. This PR fixes that. 

**The issue generated by PR #16501:**
before

**After this PR:**
after

### CODE REVIEW

File: client/less/main.less [hunk 1]
```diff
@@ -822,6 +822,11 @@ code {
       .btn-danger {
         background: #FFFFFF;
       }
+      .btn-danger:hover,
+      .btn-danger:focus,
+      .btn-danger:active {
+      background: #d9534f !important;
```
Comment (BerkeleyTrue) [inline client/less/main.less]: You're missing an indent here

Comment (vkWeb) [reply client/less/main.less]: Oh right.

**Comment by vkWeb** [comment]
@BerkeleyTrue PR updated as per your request.

I'm sorry for my silly mistake but actually, the extension I use to format my Less documents doesn't work with freeCodeCamp's Less files. Does CSS formatters work for you?

**Comment by BerkeleyTrue** [comment]
> Does CSS formatters work for you?

I use nvim and it has a build in syntax support for less. It works adequately enough.

**Comment by BerkeleyTrue** [comment]
@vkWeb fixed the issue.

Happy Coding

**Comment by vkWeb** [comment]
@BerkeleyTrue Okay. 

Thanks for the merge 😃.

**Pull request merged**